pkg/collector/corechecks/snmp/metadata/payload.go

```diff
@@ -4,6 +4,16 @@ package metadata
 // Resources are devices, interfaces, etc
 var PayloadMetadataBatchSize = 100
 
+// DeviceStatus enum type
+type DeviceStatus int32
+
+const (
+	// DeviceStatusReachable means the device can be reached by snmp integration
+	DeviceStatusReachable = DeviceStatus(1)
+	// DeviceStatusUnreachable means the device cannot be reached by snmp integration
+	DeviceStatusUnreachable = DeviceStatus(2)
+)
+
```

Comment:
And using iota:

`const (
	_ DeviceStatus = iota
    // DeviceStatusReachable means the device can be reached by snmp integration
	DeviceStatusReachable
    // DeviceStatusUnreachable means the device cannot be reached by snmp integration
	DeviceStatusUnreachable
)`

Reply:
I think it's a bit more explicit in this case :)